docs/_partner_api/endpoints.md

```diff
@@ -208,7 +208,7 @@ curl -H "Authorization: Bearer {token}" \
   -L "https://api.zapier.com/v1/zap-templates?client_id=${client_id}&limit=20&offset=20"
 ```
 
-## GET /v1/zap-templates/me
+## GET /v1/zap-templates/me [DEPRECATED ON JULY 5, 2021]
```

Comment:
I'd just delete this section so it's not in the docs at all.

Reply:
Docs for this endpoint are also on this page: https://platform.zapier.com/partners/zap-templates

https://github.com/zapier/visual-builder/blob/master/docs/_partners/zap-templates.md#my-zap-templates

Reply:
I removed those sections. 👍